docs/guides/software-setup/software-windows.md

```diff
@@ -0,0 +1,353 @@
+  ![Updating the DocumentRoot value](../../images/software-setup/windows/xampp-document-root.png){ width="600" }
+</figure>
+
+7. Save your changes and close this file. Then, click the "Stop" button next to Apache. After it **fully shuts downs**, click "Start" again.
+
+<figure markdown="span">
+  ![Stopping and starting XAMPP](../../images/software-setup/windows/xampp-stop-start.png){ width="600" }
+</figure>
+
+!!! info 
+    Apache may fail to start if the folder `C:/xampp/htdocs/lorekeeper/public` does not exist. **This is OK.** Continue with this guide, and Apache will start after you have cloned LK.
+
+8. Next, we are going to make one more change in anticipation of installing Lorekeeper. Click the "Admin" button next to MySQL.
+
+<figure markdown="span">
+  ![pressing the mysql admin button](../../images/software-setup/windows/mysql-admin.png){ width="600" }
+</figure>
+
+9. This will open a window similar to this in your browser. This is PHPMyAdmin, and it is where we control how most of the data is stored for Lorekeeper. Click "New".
+
+<figure markdown="span">
+  ![phpmyadmin front page](../../images/software-setup/windows/phpmyadmin-start.png){ width="600" }
+</figure>
+
+10. Type in `lorekeeper` or any other easy to remember name, then click "Create".
+
+<figure markdown="span">
+  ![creating a database in phpmyadmin](../../images/software-setup/windows/phpmyadmin-create.png){ width="600" }
+</figure>
+
+11. **Congratulations! We're done here for now.** Next, we will install the Git software needed to manage our Lorekeeper files.
+
+## Installing Git Software
+
+Historically, the software used for managing Git was Sourcetree, but I will be using **SourceGit** as it is more stable and frequently updated.
+
+### Installing Git
+
+1. First, we need to install Git itself. Git is a tool used to manage files and file history for coding projects. Go [here](https://git-scm.com/downloads) to download the latest copy of Git.
+
+Click this button...
+<figure markdown="span">
+  ![downloading git from the website](../../images/software-setup/windows/git-dl-1.png){ width="600" }
+</figure>
+
+..and then this link.
+<figure markdown="span">
+  ![clicking the second download link for git](../../images/software-setup/windows/git-dl-2.png){ width="600" }
+</figure>
+
+2. After the file is done downloading, run it. You will see this screen. Press "Next".
+
+<figure markdown="span">
+  ![git gnu license agreement](../../images/software-setup/windows/git-install.png){ width="600" }
+</figure>
+
+3. You may see a screen asking about a start menu folder. The default location (often "Git") is acceptable. Press "Next".
+
+4. **Make sure the highlighted fields are selected.** Press "Next".
+
+<figure markdown="span">
+  ![a checklist of installation settings for git](../../images/software-setup/windows/git-install-1.png){ width="600" }
+</figure>
+
+5. This next option is up to you. Notepad++ (if you have it installed) or Notepad are both valid options.
```

Comment:
If we added a vscode installation in before this, would it show up in the dropdown? I feel like at this point that may be best. I tend to pretty quickly insist people asking questions in the server pickup vscode if they haven't already.